packages/ui-components/src/components/calendar-advanced-date-selector/calendar-advanced-date-selector.tsx

```diff
@@ -129,12 +134,18 @@ export class KvCalendarAdvancedDateSelector implements ICalendarAdvancedDateSele
 		return undefined;
 	};
 
-	public getStartInputConfig = (): Partial<ITextField> => {
-		return merge({}, DEFAULT_START_DATE_INPUT_CONFIG, this.absoluteTimeConfig?.startInputConfig ?? {}, { placeholder: this.getAbsoluteTimeStartPlaceholder() });
+	public getStartInputConfig = (): IInputConfig => {
+		return merge({}, DEFAULT_START_DATE_INPUT_CONFIG, this.absoluteTimeConfig?.startInputConfig ?? {}, {
+			placeholder: this.getAbsoluteTimeStartPlaceholder(),
+			dateMask: this.absoluteTimeConfig?.startInputConfig?.dateMask
```

Comment:
I don't think this is needed because this object will be merged with the  merge with `this.absoluteTimeConfig?.startInputConfig ?? {}`. Please confirm it.

Reply:
Yep, that's correct. Thanks :muscle: